assets/js/theme/common/carousel/utils/tooltipSetup.js

```diff
@@ -1,23 +1,11 @@
-const carouselTooltipClass = 'carousel-tooltip';
-const carouselTooltip = `<span class="${carouselTooltipClass}"></span>`;
-
-const setupTooltipAriaLabel = ($node) => {
-    const $existedTooltip = $node.find(`.${carouselTooltipClass}`);
-
-    if ($existedTooltip.length) {
-        $existedTooltip.attr('aria-label', $node.attr('aria-label'));
-    } else {
-        const $tooltip = $(carouselTooltip).attr('aria-label', $node.attr('aria-label'));
-        $node.append($tooltip);
-    }
-};
+import setupAriaAttributes from './setupAriaAttributes';
 
 const setupArrowTooltips = (...arrowNodes) => {
-    arrowNodes.forEach($arrow => setupTooltipAriaLabel($arrow));
+    arrowNodes.forEach($arrow => setupAriaAttributes($arrow, 'aria-label', $arrow.attr('aria-label'), 'carousel-tooltip'));
 };
 
 const setupDotTooltips = ($dots) => {
-    $dots.children().each((idx, dot) => setupTooltipAriaLabel($(dot).find('button')));
+    $dots.children().each((idx, dot) => setupAriaAttributes($(dot).find('button'), 'aria-label', $(dot).find('button').attr('aria-label'), 'carousel-tooltip'));
```

Comment:
Are we able to specify selector for dot button?

Reply:
There are no specific classes for buttons and I didn’t see that these btns are being used elsewhere. if you find this more appropriate I can add such class and use them instead

Reply:
I will add class="js-carousel-dot" for button

Reply:
added